### ISSUE TITLE
Deployment issue - Microsoft flows

### ISSUE THREAD

**Issue by Farff** [OfficeDev/microsoft-teams-apps-requestateam#281]
During the deployment, when I get to step 4 and configure the Microsoft Flows, after uploading the zip file, I only have 1 flow (there should be 2 according to the documentation).  I can only see Team Request Approval flow, and the Check Team Availability flow does not show.  I've tried to upload three times and it still won't show

**Comment by OliverR-82** [comment]
Yep, I can confirm. The "Check Team Availability" flow is missing from the package. The app depends on the result of that flow, specifically you won't be able to click on 'Next' after filling in a name, description and business reason for your team.

**Comment by alexc-MSFT** [comment]
@Farff @OliverR-82 , you will find this flow in the Power Apps portal > Flows. Let me know if you find it in there.

**Comment by OliverR-82** [comment]
> @Farff @OliverR-82 , you will find this flow in the Power Apps portal > Flows. Let me know if you find it in there.

Hi Alex, Thank you for your reply. However, the flow is not listed there. The other flow - Team Request Approval - is listed, but not the 'Check Team Availability'.
![image](https://user-images.githubusercontent.com/90913691/134804365-fa24e91f-a9f1-4842-bd7a-69e994af7f72.png)
Also, when I look at the contents of the downloaded .zip file, the missing flow is also not included within the file.

**Comment by alexc-MSFT** [comment]
@OliverR-82 are you still stuck with this?
